README.md

```diff
@@ -29,7 +29,7 @@ Download source code tarballs:
 http://rawtherapee.com/shared/source/
 
 Git handbook:
-http://git-scm.com/book/en/
+https://git-scm.com/book/en/v2
```

Comment:
Hello
Changing to https should be fine, but please remove the /v2. There is no need for it, and it points users to an old version when v3 comes along.